docs/user/index.md

```diff
@@ -134,10 +130,10 @@ status.
 
 ### Organization webhooks
 
-GitHub Organization members that have the [Owner role][owners], are allowed
+GitHub organization members that have the [owner role][owners], are allowed
```

Comment:
GitHub refers to this role as "Owner", not "owner". For example, https://github.com/orgs/atomist/people